pages/contract_development_and_manufacturing_organization.tsx

```diff
@@ -0,0 +1,86 @@
+                    <li>We have the expertise and resources necessary to address different development stages that occur before the drug manufacturing stage.</li>
+                    <li>Our team possesses the potential to handle the design, production and examination stages of early drug creation.</li>
+                    <li>Outsourcing your drug manufacturing tasks to Diversity Bio Sciences will give you access to cutting-edge resources and technology without investing a huge amount of money and time towards developing the right in-house infrastructure.</li>
+                </ul>
+            </div>
+
+            <div>
+                <h2 className="sub-heading">Diversity Bio Sciences: Manufacturing Pharmaceuticals While Ensuring Success, Safety and Compliance</h2>
+                <p>
+                Serving as the reliable partner for several generic companies and leading innovators and having a proven track record on chemical process development & manufacturing, we have also ventured into Contract Research and Manufacturing Services (CRAMS). With unwavering dedication to state-of-the-art and quality research; intellectual and regulatory property expertise and universal manufacturing potentials, we are all set to support and speed up the manufacturing and manufacturing programs of several innovator companies.
+                </p>
+                <p>
+                At Diversity Bio Sciences, our 6 years of experience is judiciously employed to generate commercially viable, top quality, scalable and affordable procedures for all our partners. Want to experience our world-class scientific infrastructure and community? Get in touch with us at +91 9948872227 or mail us at <a href="mailto:sales@diversitybiosciences.com">sales@diversitybiosciences.com.</a>.
+                </p>
+            </div>
+
+            <div>
+                <h3 className="sub-heading">Frequently Asked Questions (FAQs)</h3>
+                <div className="faqsection">
+                    <h4>Is contract research and custom synthesis a service only meant for the large pharmaceutical companies?</h4>            
+                    <p>
+                    No, not at all. Such services are not meant for any individual industry or a set group of customers. As a CRO, Diversity Bio Sciences serves pharmaceutical companies of all sizes without differentiating between varied customer branches.
+                    </p>
+                </div>
+
+                <div className="faqsection">
+                    <h4>How can I benefit from outsourcing research activities to Diversity Bio Sciences? </h4>            
+                    <p>
+                    Outsourcing research activities to us means major improvement in product development. Other benefits of our services include saved costs, time and resources.
+                    </p>
+                </div>
+
+                <div className="faqsection">

```

Comment:
crreate a faqsection component which takes title and text and render it and use that instead of duplicating the div tags